conformance-test/env.sh

```diff
@@ -9,11 +9,11 @@ export KEPPEL_DB_CONNECTION_OPTIONS=sslmode=disable
 export KEPPEL_DB_PASSWORD=
 export KEPPEL_DB_PORT=54321
 
+KEPPEL_FILESYSTEM_PATH=./conformance-test/storage
```

Comment:
Why not export?

Reply:
Because I don't want to create the impression that Keppel reads this variable. This is only used to DRY the shell script.